discoverable/README.md

```diff
@@ -0,0 +1,30 @@
+## How to craft discoverable repos
+
+For your repositories to flourish, it's important that people know they exist! Here are some tips for crafting discoverable repositories. 
+
+### Repository purpose
+- Make sure you solve a real problem. Repositories that have useful, helpful information will always rise to the top. 
+- Make sure your idea behind the solution and your vision are clearly written down. It helps if people know not only what the repository is _for_, but where it came from and where it's going. 
+
+### Sharing 
+- Tell users to star your repo if they find it interesting.
+- If you have a blog or mailing list, use it. Once your repository is ready, send an announcement your subscribers.
```

Comment:
"If you have a blog... send an announcement **to** your subscribers."

- missing a word